### ISSUE TITLE
What's the difference between pretrained model: ESC_XL_DF2K_X2_FT.pth and ESC_XL_DF2K_X2_PT.pth? What's the "FT" and "PT" stands for?

### ISSUE THREAD

**Issue by tomcan2015** [dslisleedh/ESC#14]

**Comment by dslisleedh** [comment]
Thank you for your interest, @tomcan2015 !

Following ATD (CVPR`24), we first pre-trained ESC-XL for 300K iterations using 64x64 input patches and then fine-tuned it for 250K iterations using 96x96 input patches.
So, PT and FT mean pre-training and fine-tuning, respectively.

Please let me know if you have any further questions.

**Comment by dslisleedh** [comment]
You’re spot on, @tomcan2015!

Variants named without "real" were trained only on bicubic-downsampled LR/HR pairs, so they can be unstable on images with real-world degradations (noise, compression artifacts, blur). 
Following [Real-ESRGAN](https://openaccess.thecvf.com/content/ICCV2021W/AIM/papers/Wang_Real-ESRGAN_Training_Real-World_Blind_Super-Resolution_With_Pure_Synthetic_Data_ICCVW_2021_paper.pdf?utm_source=chatgpt.com), we trained  “real” variants with LR/HR pairs generated by a high-order degradation pipeline (various types/levels of noise, blur, downsampling, and compression), which is why they’re more robust on compressed inputs.
However, the models trained with pixel-wise loss, such as MSE and L1, tend to reconstruct over-smoothed outputs. 
Many SR methods, therefore, improve perceptual quality by fine-tuning with a GAN loss, and this is where our ESC_Real_XL_GAN fails.
I think, for large-sized real-world SR models, diffusion-based methods are better.

Regarding your question: the “real” variants currently support 4× upscaling only.

**Comment by tomcan2015** [comment]
Thanks again! Your different models are very impressive and showing different bias in today's image restoration field. The large kernel design has some advantage on real HW as the efficiency of GPU or NPU increases with the kernal size. 
A blending between MSE and GAN may be a good choice at this moment for real time SR. Diffusion method takes too long time today.
One suggestion is to include "Defocus" area consideration into image restoration. You may refer to this paper: DAQE: Enhancing the Quality of Compressed
Images by Exploiting the Inherent
Characteristic of Defocus
Qunliang Xing, Graduate Student Member, IEEE, Mai Xu, Senior

I usually find SR models try to enhance the "Defocus" area as well especially those GAN models. It actually add something we don't expect or something too sharp in the "Defocus" area. It actually makes the image become unreal and we just want the "Defocus" area keep blurred. For these area, simply remove noise and compression artifacts (like blockiness) is good enough. We don't want to output an image with sharp details everywhere. The "Defocus" area is intentionally left blur by the photographier.

**Comment by dslisleedh** [comment]
Thank you for your suggestion, @tomcan2015 !
I agree that addressing over-sharpening the defocused area is important for real-world applications. 
One possible way that can be considered now is to train a model with mild degradations. 
Specifically, I think removing the blur operations from the degradation pipeline or using a one-stage degradation pipeline can reduce the over-sharpening problem. 
Since the current RealESRGAN degradation pipeline generates significantly degraded low-quality inputs for training, the model might not perceive the defocused area and try to remove any blurriness in the given images.
